### ISSUE TITLE
UnloadedDataException: Styles not yet  loaded

### ISSUE THREAD

**Issue by fox-mali** [dvanoni/notero#191]
Question:
When saving a project to Zero, notion cannot create an entry normally
<img width="192" alt="4eb82c327f8a5183fab4451524951f4" src="https://user-images.githubusercontent.com/57664779/207590395-3a1fc635-b790-4037-ae8f-3384a85fb01d.png">
Here are pictures that may help for  finding the problem:
Screenshots:
![image](https://user-images.githubusercontent.com/57664779/207590754-2939cc07-1b14-4933-91fd-dbc41b4a8c60.png)
![image](https://user-images.githubusercontent.com/57664779/207590826-d813d240-980c-4dde-bbd7-49eec9ac0229.png)
![image](https://user-images.githubusercontent.com/57664779/207590905-e8e01ec0-0818-4ef7-8b98-55198ba9fe31.png)
![image](https://user-images.githubusercontent.com/57664779/207590967-4f40b2d4-2138-4387-96b5-297430d8e509.png)
I hope you can give an solution， thank you very much!

**Comment by dvanoni** [comment]
Hi, @fox-mali. Thanks for reporting this and sharing the screenshots!

I haven't seen this error before. I looked into the Zotero code a bit, and it looks like this is happening because the citation styles have not loaded for some reason. I'm not sure why this is, but I wonder if the fourth message down is related:

> Error(s) encountered during statement execution: data disk image is malformed

I looked into this in the Zotero code as well, and I found a reference to this support page: https://www.zotero.org/support/kb/corrupted_database

Is there anything in that page that applies to you and can maybe help?
